app/presenters/api/admin/green_lanes/category_assessment_presenter.rb

```diff
@@ -0,0 +1,15 @@
+module Api
+  module Admin
+    module GreenLanes
+      class CategoryAssessmentPresenter < WrapDelegator
+        def green_lanes_measure_ids
+          green_lanes_measures.map(&:id)
+        end
+
+        def exemption_ids
+          exemptions.map(&:id)
+        end
+      end
+    end
+  end
+end
```

Comment:
Is this needed? Doesn't the `association_pks` plugin give you these? That was why I'd included that originally?

Reply:
OK, `association_pks` plugin adds id method with `_pks` sufix not `_ids` suffix. This can be done with `id_method_name` added to the serializer. This is done now and I removed the presenter now.